data/Makefile

```diff
@@ -686,6 +686,11 @@ minsize=2000
 ################################################################################
 # Physlr
 
+# Generate repetitive Bloom filter to select minimizers
+bloom_filter_size=10000000000 #10G
```

Comment:
I think we should put this with the other declared variables at the top of the Makefile